voip_utils/voip.py

```diff
@@ -175,6 +178,7 @@ def datagram_received(self, data, addr):
         if not self._is_connected:
             return
 
+        _LOGGER.debug("Datagram received on addr %s, current addr is %s", addr, self.addr)
         if self.addr is None:
```

Comment:
Should we just always set address?

Reply:
That would probably work as a stopgap fix to get out quickly, but I think a more technically correct solution would require some broader changes.

I suspect The VoipAssistSatellite should not be extending RtpDatagramProtocol as a new RtpDatagramProtocol should be getting created for each call, but the VoipAssistSatellite object has a much longer lifetime. It seems we can not assume a VOIP device is going to reuse the same RTP address/port. Based on this comment https://github.com/home-assistant/core/issues/128372#issuecomment-2564718981 I think we should be initializing the RtpDatagramProtocol with the host and port from the call_info instead of waiting to set it in the datagram_received method.